app/src/main/java/com/example/boylc/demomvp/data/model/Category.java

```diff
@@ -0,0 +1,74 @@
+package com.example.boylc.demomvp.data.model;
+
+import com.example.boylc.demomvp.untils.Constants;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Category {
+    private int mId;
+    private String mCategoryName;
```

Comment:
Các biến này đều là instace variable của class Category nên e k cần đặt tiền tố Category ở đầu làm gì cả
```suggestion
    private String mName;
```

Reply:
tuơng tự cho các biến khác